assets/models/system/bytetrack_yolox_x_crowdhuman_mot17-private-half/spec.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: bytetrack_yolox_x_crowdhuman_mot17-private-half
+path: ./
+properties:
+  finetune-min-sku-spec: 4|1|28|176
+  finetune-recommended-sku: Standard_NC6s_v3
+  finetuning-tasks: video-multi-object-tracking
+  inference-min-sku-spec: 2|1|14|28
+  inference-recommended-sku: Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
+    Standard_NC24rs_v3, Standard_NC16as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,
+    Standard_NC4as_T4_v3, Standard_NC64as_T4_v3, Standard_NC8as_T4_v3, Standard_NC96ads_A100_v4,
+    Standard_ND40rs_v2, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
+  model_id: yolof_r50_c5_8x8_1x_coco
+tags:
+  Preview: ''
+  license: apache-2.0
+  model_specific_defaults:
+    apply_deepspeed: 'false'
+    apply_ort: 'false'
+  task: video-multi-object-tracking
```

Comment:
Is this that Rashuad suggested us to use ?

Reply:
ohhhh I thought this should go with the model. thanks for the point out!